operations/alloy-mixin/alerts.libsonnet

```diff
@@ -1,9 +1,20 @@
+local clusterAlerts = (import './alerts/clustering.libsonnet');
+local controllerAlerts = (import './alerts/controller.libsonnet');
+local openTelemetryAlerts = (import './alerts/opentelemetry.libsonnet');
+
 {
   prometheusAlerts+: {
-    groups+: [
-      (import './alerts/clustering.libsonnet'),
-      (import './alerts/controller.libsonnet'),
-      (import './alerts/opentelemetry.libsonnet'),
-    ],
+    groups+: 
+    if $._config.enableAlloyCluster then
+      [      
+        clusterAlerts.newAlloyClusterAlertsGroup($._config.enableK8sCluster),
+        controllerAlerts.newControllerAlertsGroup($._config.enableK8sCluster),
+        openTelemetryAlerts.newOpenTelemetryAlertsGroup($._config.enableK8sCluster),
+      ]
+      else         
+      [
+        controllerAlerts.newControllerAlertsGroup($._config.enableK8sCluster),
+        openTelemetryAlerts.newOpenTelemetryAlertsGroup($._config.enableK8sCluster)
+      ],
   },
```

Comment:
Maintenance wise, I'm worried about having two arrays where someone might forget to put a new alert group on both. Can we change this to be additive similarly to how `dashboards.libsonnet` has "otherDashboards" and "alloyClusterDashboards"?